app/services/forest_liana/search_query_builder.rb

```diff
@@ -15,11 +15,10 @@ def initialize(params, includes, collection)
     def perform(resource)
       @resource = @records = resource
       @tables_associated_to_relations_name =
-        ForestLiana::QueryHelper.get_tables_associated_to_relations_name(@resource)
+          ForestLiana::QueryHelper.get_tables_associated_to_relations_name(@resource)
```

Comment:
Bad indentation all over the PR (Maybe your text editor is not well configured). Would you mind changing it back to how it was ? It will ease the review later on.